pkg/api/controllers/swagger.yaml

```diff
@@ -1284,6 +1377,24 @@ components:
         next:
           type: string
           example: ""
+    Log:
+      type: object
+      properties:
```

Comment:
Can you add "required" key with all required fields?
Without this, generated SDK come with pointers on fields instead of scalar types.